packages/data-explorer/tests/unit/store/mutations.spec.ts

```diff
@@ -51,35 +52,8 @@ describe('mutations', () => {
   let baseAppState: ApplicationState
 
   beforeEach(() => {
-    baseAppState = {
-      toast: null,
-      settingsTable: 'de_dataexplorer_table_settings',
-      tableName: 'root_hospital_patients',
-      tableData: null,
-      tableMeta: null,
-      dataDisplayLayout: 'CardView',
-      dataDisplayLimit: 100,
-      defaultEntityData: null,
-      showShoppingCart: false,
-      shoppedEntityItems: [],
-      tableSettings: {
-        settingsRowId: null,
-        collapseLimit: 5,
-        customCardCode: null,
-        customCardAttrs: '',
-        isShop: false,
-        defaultFilters: []
-      },
-      filters: {
-        hideSidebar: false,
-        definition: [],
-        shown: [],
-        selections: {}
-      },
-      searchText: '',
-      bookmark: '',
-      componentRoute: false
-    }
+    baseAppState = mockState()
```

Comment:
Personally i am not a fan of importing a general mock into multiple test, as the creates interdependencies that would otherwise not be needed. In addition mocking more than needed for the test makes is harder to read and debug the test. The need for this kind of mock signals a dependency that probably needs to be refactored to be less interdependent . As a alternative you could create a mockstate in de test `state:any = { my-prop: 'needed-in-test'}`.